commit fb4d1d10f3bcd0bff63658b485da09776a2cc383
Author: Sven Eden <sven@eden-worx.com>
Date:   Mon Jan 13 12:32:46 2025 +0100

    elogind doesn't allocate VTs, so do not change those setup by login
    
    Signed-off-by: Sven Eden <sven@eden-worx.com>

diff --git a/src/basic/terminal-util.c b/src/basic/terminal-util.c
index 6b54dd18e..2d619ccfd 100644
--- a/src/basic/terminal-util.c
+++ b/src/basic/terminal-util.c
@@ -1487,12 +1487,14 @@ int vt_restore(int fd) {
                         q = -errno;
         }
 
+#if 0 /// elogind doesn't allocate VTs and shouldn't change those setup by login(1).
         r = fchmod_and_chown(fd, TTY_MODE, 0, GID_INVALID);
         if (r < 0) {
                 log_debug_errno(r, "Failed to chmod()/chown() VT, ignoring: %m");
                 if (q >= 0)
                         q = r;
         }
+#endif // 0
 
         return q;
 }
